utils.js

```diff
@@ -1,5 +1,10 @@
+const axios = require('axios');
+const createError = require('http-errors');
 const { cloneDeep } = require('lodash');
+const ERROR = require('./constants/error');
 const GIT = require('./constants/git');
+const { AT_SIGN_END, AT_SIGN_BEGIN } = require('./constants/stringProcessing');
+const STRING_PROCESSING = require('./constants/stringProcessing');
```

Comment:
```suggestion
const axios = require('axios');
const createError = require('http-errors');
const { cloneDeep } = require('lodash');

const ERROR = require('./constants/error');
const GIT = require('./constants/git');
const { AT_SIGN_END, AT_SIGN_BEGIN } = require('./constants/stringProcessing');
const STRING_PROCESSING = require('./constants/stringProcessing');
```
성향 차이적인 부분이지만 외부 모듈과 내부 파일 간에는 한줄 띄워서 구분하는게 보기 더 좋을 것같아요.